lib/components/viewers/nearby/nearby-view.tsx

```diff
@@ -286,6 +306,32 @@ function NearbyView({
       >
         {/* This is used to scroll to top */}
         <div aria-hidden ref={firstItemRef} />
+        <LocationField
+          className="nearby-view-location-field"
+          geocoderConfig={geocoderConfig}
+          getCurrentPosition={getCurrentPosition}
+          inputPlaceholder={intl.formatMessage({
+            id: 'components.NearbyView.searchNearby'
+          })}
+          location={{
+            // Provide a 0 default in case the nearby view coords are null
+            lat: 0,
+            lon: 0,
+            ...nearbyViewCoords,
+            name: reversedPoint
+          }}
+          LocationIconComponent={() => (
```

Comment:
Hmm this isn't showing up on chrome or safari
![image](https://github.com/user-attachments/assets/f9c5743f-3cee-4b86-876a-4104e9c3dc30)

Reply:
Also, when it does show up it feels v close to the placeholder text? Probably better suited to a OTP-UI PR, but I guess just flagging it here

![image](https://github.com/user-attachments/assets/0395a52a-a365-48c5-aae4-2cf5efd6ecb7)

Reply:
Yeah these might both be otp-ui issues!